services/galley/test/integration/API/Util/TeamFeature.hs

```diff
@@ -150,32 +149,50 @@ getAllFeatureConfigsWithGalley galley uid = do
       . paths ["feature-configs"]
       . zUser uid
 
+putTeamFeatureFlagWithGalley ::
+  forall (a :: Public.TeamFeatureName).
+  ( HasCallStack,
+    Public.KnownTeamFeatureName a,
+    ToJSON (Public.TeamFeatureStatus 'Public.WithoutLockStatus a)
+  ) =>
+  (Request -> Request) ->
+  UserId ->
+  TeamId ->
+  Public.TeamFeatureStatus 'Public.WithoutLockStatus a ->
+  TestM ResponseLBS
+putTeamFeatureFlagWithGalley galley uid tid status =
```

Comment:
you could have implemented this in terms of `putTeamFeatureFlagInternal` below, but we want to get rid of all this code anyway and generate it using servant.  so let's keep this, it certainly looks like it works.

Reply:
Hm, but don't we want to test that the authorized team owner can do this? If we use the internal endpoint, that doesn't get tested, instead we test that customer support can change this via the internal API, right?

Reply:
ah, you're right, my bad.